tests/test-module-for-component-test.js

```diff
@@ -114,65 +114,6 @@ test('can lookup components in its layout', function() {
   equal(component._state, 'inDOM');
 });
 
-test('can lookup array controllers in its layout', function() {
-  expect(1);
-  var component = this.subject({
-    colors: ['red', 'green', 'blue'],
-    layout: Ember.Handlebars.compile("{{#each colors itemController='color'}}{{hexa}}{{/each}}")
-  });
-  this.render();
-  equal(component._state, 'inDOM');
-});
-
-test('can lookup object controllers in its layout', function() {
-  expect(1);
-  var component = this.subject({
-    colors: ['red', 'green', 'blue'],
-    layout: Ember.Handlebars.compile("{{#each colors itemController='object'}}{{this}}{{/each}}")
-  });
-  this.render();
-  equal(component._state, 'inDOM');
-});
-
-test('can lookup basic controllers in its layout', function() {
-  expect(1);
-  var component = this.subject({
-    colors: ['red', 'green', 'blue'],
-    layout: Ember.Handlebars.compile("{{#each colors itemController='basic'}}{{this}}{{/each}}")
-  });
-  this.render();
-  equal(component._state, 'inDOM');
-});
-
-test('can lookup Ember.Select in its layout', function() {
-  expect(1);
-  var component = this.subject({
-    colors: ['red', 'green', 'blue'],
-    layout: Ember.Handlebars.compile("{{view 'select'}}")
-  });
-  this.render();
-  equal(component._state, 'inDOM');
-});
-
-test('can lookup default Ember.Views in its layout', function() {
```

Comment:
We should replace this one with a {{component}} helper version (not blocking this PR though).